mmv1/products/compute/BackendService.yaml

```diff
@@ -264,6 +262,34 @@ properties:
           description: |
             Used when balancingMode is UTILIZATION. This ratio defines the
             CPU utilization target for the group. Valid range is [0.0, 1.0].
+        - !ruby/object:Api::Type::Boolean
+          name: failover
+          description: |-
+            This field designates whether this is a failover backend. More than one
+            failover backend can be configured for a given BackendService.
+        - !ruby/object:Api::Type::Enum
+          name: preference
+          description: |
+            This field indicates whether this backend should be fully utilized
+            before sending traffic to backends with default preference. The possible
+            values are: - PREFERRED: Backends with this preference level will be
+            filled up to their capacity limits first, based on RTT. - DEFAULT: If
+            preferred backends don't have enough capacity, backends in this layer
+            would be used and traffic would be assigned based on the load balancing
+            algorithm you use. This is the default
+            The possible values are:
+
+            * `DEFAULT`: No preference.
+
+            * `PREFERENCE_UNSPECIFIED`: If preference is unspecified, we set it to
+                                       the DEFAULT value
+
+            * `PREFERRED`: Traffic will be sent to this backend first.
+          values:
+            - :DEFAULT
+            - :PREFERENCE_UNSPECIFIED
```

Comment:
We probably don't want to allow a user to specifically set the _UNSPECIFIED value, as APIs tend to reject that value. It's only used (and not returned by the API) if the value is not sent by the client

Reply:
Hi @slevenick , the change to the backendService accidentally was included in this pr in the second commit. It should not have been. My apologies.